### DISCUSSION TITLE
Pin important issues

### DISCUSSION THREAD

**Discussion by larsbarring** [cf-convention/discuss#345] [Comments and ideas for changing CF]
### Topic for discussion

Github allows up to three issues to be pinned at the top of the issue list page for each repository, as well as the Discussions. II think this can be useful, a couple of examples:
* *Vocabularies* repo https://github.com/cf-convention/vocabularies/issues/10
* Here in *Discussions* announcements could be pinned, and then removed when outdated, e.g. https://github.com/orgs/cf-convention/discussions/342
* Permanently pinned issues may of course point to Code of Conduct and so forth ...

**Comment by DocOtak** [comment]
Re Code of Conduct, I would recommend this be added to the host repository for these discussions under the .github directory. See [how to add](https://docs.github.com/en/enterprise-cloud@latest/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project) and [the github guide/recommendations](https://docs.github.com/en/enterprise-cloud@latest/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project). Doing this will basically add a "code of conduct" link to many places in the discussions feature and not just pinned to the top.

I support pinning that Workshop discussion since it has a time dependency.
